worlds/pokemon_rb/docs/setup_fr.md

```diff
@@ -0,0 +1,102 @@
+# Guide de configuration pour Pokémon Rouge et Bleu : Archipelago
+
+## Important
+
+Comme nous utilisons BizHawk, ce guide s'applique uniquement aux systèmes Windows et Linux.
+
+## Logiciel requis
+
+- BizHawk : [Version BizHawk de TASVideos](https://tasvideos.org/BizHawk/ReleaseHistory)
+   - Les versions 2.3.1 et ultérieures sont prises en charge. La version 2.9.1 est recommandée.
+   - Des instructions d'installation détaillées pour BizHawk peuvent être trouvées sur le lien ci-dessus.
+   - Les utilisateurs Windows doivent d'abord exécuter le programme d'installation des prérequis, qui peut également être trouvé sur le lien ci-dessus.
+- Le client Archipelago intégré, qui peut être installé [ici](https://github.com/ArchipelagoMW/Archipelago/releases)
+   (sélectionnez `Pokemon Client` lors de l'installation).
```

Comment:
```suggestion 
```
When installing archipelago, you do not have the option to select specific client anymore.